pkg/printers/internalversion/printers_test.go

```diff
@@ -5758,6 +5758,153 @@ func TestPrintLease(t *testing.T) {
+		}
+		return res
+	}
+
+	tests := []struct {
+		name     string
+		options  printers.GenerateOptions
+		lr       api.LimitRange
+		expected []metav1.TableRow
+	}{
+		{
+			lr: api.LimitRange{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "test-limitrange",
+					Namespace:         "default",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: api.LimitRangeSpec{
+					Limits: []api.LimitRangeItem{
+						{
+							Type:           api.LimitTypeContainer,
+							Max:            getResources("100m", "10000m", "", ""),
+							Min:            getResources("0m", "100m", "", ""),
+							DefaultRequest: getResources("10m", "100m", "", ""),
+						},
+					},
+				},
+			},
+			expected: []metav1.TableRow{{Cells: []interface{}{"test-limitrange", string("0s")}}},
+		},
+		{
+			options: printers.GenerateOptions{Wide: true},
+			lr: api.LimitRange{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "test-limitrange",
+					Namespace:         "default",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: api.LimitRangeSpec{
+					Limits: []api.LimitRangeItem{
+						{
+							Type:           api.LimitTypeContainer,
+							Max:            getResources("100m", "10000m", "", ""),
+							Min:            getResources("0m", "100m", "", ""),
+							DefaultRequest: getResources("10m", "100m", "", ""),
+						},
+					},
+				},
+			},
+			expected: []metav1.TableRow{{Cells: []interface{}{"test-limitrange", string("0s"), string("Container"), string("cpu=0, memory=100m"), string("cpu=100m, memory=10")}}},
+		},
+		{
+			options: printers.GenerateOptions{Wide: true},
+			lr: api.LimitRange{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "test-limitrange",
+					Namespace:         "default",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: api.LimitRangeSpec{
+					Limits: []api.LimitRangeItem{
+						{
+							Type:                 api.LimitTypeContainer,
+							Max:                  getResources("100m", "10000Mi", "", ""),
+							Min:                  getResources("0m", "100Mi", "", ""),
+							Default:              getResources("50m", "500Mi", "", ""),
+							DefaultRequest:       getResources("10m", "200Mi", "", ""),
+							MaxLimitRequestRatio: getResources("10", "", "", ""),
+						},
+						{
+							Type:           api.LimitTypePod,
+							Max:            getResources("500m", "50000Mi", "", ""),
+							Min:            getResources("100m", "5000Mi", "", ""),
+							Default:        getResources("50m", "500Mi", "", ""),
+							DefaultRequest: getResources("10m", "200Mi", "", ""),
+						},
+					},
+				},
+			},
+			expected: []metav1.TableRow{{Cells: []interface{}{"test-limitrange", string("0s"), string("Container"), string("cpu=0, memory=100Mi"), string("cpu=100m, memory=10000Mi"), string("Pod"), string("cpu=100m, memory=5000Mi"), string("cpu=500m, memory=50000Mi")}}},
```

Comment:
the cells don't correctly map to the columndefinition headers when there's more than one limit